### DISCUSSION TITLE
vben v5版本 使用auto-import后会报错ESLint: 'ref' is not defined.(no-undef)

### DISCUSSION THREAD

**Discussion by linsxw** [vbenjs/vue-vben-admin#4489] [Q&A]
你们会有这样的情况吗  在vben v5版本使用auto-import插件后 不导入的话 会报错ESLint: 'ref' is not defined.(no-undef)
求助求助！！！！

**Comment by namefhf** [comment]
https://github.com/unplugin/unplugin-auto-import?tab=readme-ov-file#typescript

**Comment by elarbor** [comment]
vben5是 https://github.com/vbenjs/vue-vben-admin 吗？
